src/lib/editableGrid/Cell.tsx

```diff
@@ -90,6 +91,29 @@ export default function Cell({
     }
   }
 
+  /**
+   * This function is needed to select text in number-cells via first click on the cell
+   */
+  function onClick(e: React.MouseEvent<Element>) {
```

Comment:
> have not so easy process:
> 
> * click into cell Age
> * data was select
> * press Del
> * click into another cell Age
> * data wasn't select
> * click into 3rd cell Age
> * data was select

That occurs because of changing tabidexes of cells after every table's rerender. You pressed Del and tabindexes were changed. We haven't got any id's or static coordinates for cells yet and it's big problem for future development. When we have good way to identify cells, that problem will be fixed by couple simple changes.

Reply:
What abouto using store or el.dataset.* to identify
or as you said - el.id?

Reply:
We can add any attributes, but with next render they will be changed. We need, for example, static ids from database, or to develop static coordinates of cells